src/components/button/README.md

```diff
@@ -0,0 +1,5 @@
+## Button component
+
+```typescript
+const example: string = "My Button"
```

Comment:
As I understood, this was just an example. I suggest to put Props type here to have a quick idea about the types.

Reply:
@VLegakis you can try [this](https://github.com/netdata/netdata-ui/blob/1426-modal/src/components/sidebar/README.md) as example